.github/workflows/ci.yml

```diff
@@ -1073,11 +1147,11 @@ jobs:
         working-directory: ${{ env.HOME }}
         run: |
           sudo ln -s /usr/include/locale.h /usr/include/xlocale.h
-          if [ "${{matrix.config.version}}" = "13" ]
+          if [ "${{matrix.config.version}}" -ge "13" ]
           then
               wget https://apt.llvm.org/llvm.sh
               chmod +x llvm.sh
-              sudo ./llvm.sh 13
+              sudo ./llvm.sh "${{matrix.config.version }}"
```

Comment:
Consider removing the space before the trailing braces.

Reply:
Nice catch, thanks.